internal/integration_test.go

```diff
@@ -30,7 +30,10 @@ func TestRESTAcceptance(t *testing.T) {
 		err := cleanDB(pgDB)
 		require.NoError(t, err)
 
-		store := db.Store{Q: testQueries}
+		// FIXME I do not like this
```

Comment:
Prepending a `TODO:` on these comments will make em easier to find later. 
It would be helpful to also comment about what exactly you don't like so the next person, (or you), can remember/understand how to change it in the future
```suggestion
		// TODO: FIXME I do not like this
```

Reply:
ah I can do that, I think its because the extension I use highlights `FIXME` as red, `NOTE` as blue & `TODO` as yellow if its the first word of the comment. I should remember that others may have different extensions.

I will add more detail on why I dont like that tho

Reply:
I removed the line now, was more that I didn't like exporting the database and rather use the `New` constructor to instanciate a store. I know its just a test but helps incase we were to ever decide on moving from sqlc